ci-operator/config/openshift-kni/cnf-features-deploy/openshift-kni-cnf-features-deploy-release-4.13.yaml

```
@@ -0,0 +1,110 @@
+base_images:
+  base:
+    name: "4.13"
+    namespace: ocp
+    tag: base
+  openshift_release_golang-1.19:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+  os-min8:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "8"
```

change base image to ubi9

```suggestion
  os-min9:
    name: ubi-minimal
    namespace: ocp
    tag: "9"
```

---

I am not sure if 4.13 will ship with rhel-9 still as the testing is on going.. so I didnt make the update in this change.. once we have agreement this change can follow the branch cut.

---

I think that's still the plan of record, but it's true we're not actually there yet :(